net-libs/libtins/libtins-3.5.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils
+
+DESCRIPTION="High-level, multiplatform C++ network packet sniffing and crafting library."
+HOMEPAGE="https://libtins.github.io/"
+SRC_URI="https://github.com/mfontanini/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+cxx11 +ack-tracker +wpa2 +dot11 static-libs"
+
+REQUIRED_USE="
+	wpa2? ( dot11 )
+"
+
+DEPEND="
+	ack-tracker? ( dev-libs/boost )
```

Comment:
Most of C/C++ libraries you link to take `:=` slot operator, to ensure that your package gets rebuilt when they change SONAME.